Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.py

```diff
@@ -0,0 +1,1150 @@
+        ``args['Close_Reason']`` Alert's close reason
+        ``args['Alert_id']`` Array of alert ids to be closed
+
+    :return: Result of execution
+    :rtype: ``bool``
+
+    """
+    close_reason = args.get('Close_Reason', None)
+    close_reasons = list(filter(lambda name: name != 'None', CLOSE_REASONS.keys()))
+    if close_reason not in close_reasons:
+        raise ValueError(f'close reason must be one of {close_reasons}')
+
+    close_reason_id = CLOSE_REASONS[close_reason]
+
+    return varonis_update_alert(client, close_reason_id, ALERT_STATUSES['Closed'], argToList(args.get('Alert_id')))
+
+
+def varonis_get_alerted_events_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Get alerted events from Varonis DA
+
+    :type client: ``Client``
+    :param client: Http client
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['Alert_id']`` List of alert ids
+        ``args['page']`` Page number (default 1)
+        ``args['max_results']`` The max number of alerts to retrieve (up to 5k)
+
+    :return:
+        A ``CommandResults`` object
+
+    :rtype: ``CommandResults``
+    """
+    alerts = args.get('Alert_id', None)
+    page = args.get('page', '1')
+    max_results = args.get('max_results', '5000')
+
+    alerts = try_convert(alerts, lambda x: argToList(x))
+    max_results = try_convert(
+        max_results,
+        lambda x: int(x),
+        ValueError(f'max_results should be integer, but it is {max_results}.')
+    )
+    page = try_convert(
+        page,
+        lambda x: int(x),
+        ValueError(f'page should be integer, but it is {page}.')
+    )
+
+    result = varonis_get_alerted_events(client, alerts, max_results, page)
+    outputs = create_output(EVENT_OUTPUT, result['rows'])
+    page_size = result['rowsCount']
+    outputs = enrich_with_pagination(outputs, page, page_size)
+    return CommandResults(
+        outputs_prefix='Varonis',
+        outputs_key_field='Varonis.Event.ID',
+        outputs=outputs
+    )
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    """Main function, parses params and runs command functions
+
+    :return:
+    :rtype:
+    """
+
+    username = demisto.params().get('credentials', {}).get('identifier')
+    password = demisto.params().get('credentials', {}).get('password')
+
+    # get the service API url
+    base_url = urljoin(demisto.params()['url'], '/DatAdvantage')
+
+    # if your Client class inherits from BaseClient, SSL verification is
+    # handled out of the box by it, just pass ``verify_certificate`` to
+    # the Client constructor
+    verify_certificate = not demisto.params().get('insecure', False)
+
+    # if your Client class inherits from BaseClient, system proxy is handled
+    # out of the box by it, just pass ``proxy`` to the Client constructor
+    proxy = demisto.params().get('proxy', False)
```

Comment:
```suggestion
    params = demisto.params()
    username = params.get('credentials', {}).get('identifier')
    password = params.get('credentials', {}).get('password')

    # get the service API url
    base_url = urljoin(params['url'], '/DatAdvantage')

    # if your Client class inherits from BaseClient, SSL verification is
    # handled out of the box by it, just pass ``verify_certificate`` to
    # the Client constructor
    verify_certificate = not params.get('insecure', False)

    # if your Client class inherits from BaseClient, system proxy is handled
    # out of the box by it, just pass ``proxy`` to the Client constructor
    proxy = params.get('proxy', False)
```

Reply:
make sure to change in all the relevant places

Reply:
@yucohen fixed